packages/block-editor/src/components/block-mobile-toolbar/test/block-actions-menu.native.js

```diff
@@ -67,6 +67,7 @@ describe( 'Block Actions Menu', () => {
 		it( 'moves blocks up and down', async () => {
 			const screen = await initializeEditor( {
 				initialHtml: INITIAL_HTML,
+				screenWidth: 100,
```

Comment:
This makes the editor canvas to be smaller so the move-up/down options show in the block actions menu instead of the usual up/down arrows below the blocks, this way we can test that functionality.

<kbd><img src="https://user-images.githubusercontent.com/4885740/208931575-1a8ff9ae-256e-4153-a07b-922094cbe349.png" width="150" /></kbd>

In this example, I forced the editor to show up like that so I could show those options, it is not a real example of when the up/down buttons are collapsed.